message_ix/core.py

```diff
@@ -142,12 +122,18 @@ def cat(self, name, cat):
         """
         return ixmp.to_pylist(self._jobj.getCatEle(name, cat))
 
-    def has_solution(self):
```

Comment:
It doesn't look like this was deprecated in the last release. Is this removed entirely? How should the user check for the existence of a solution?

Reply:
The function exists as `ixmp.Scenario`, no need to duplicate.